pkg/cloudprovider/providers/gce/gce_loadbalancer.go

```diff
@@ -875,6 +937,44 @@ func (gce *GCECloud) firewallNeedsUpdate(name, serviceName, region, ipAddress st
 	return true, false, nil
 }
 
+func (gce *GCECloud) ensureHttpHealthCheckFirewall(region string, hosts []*gceInstance, hcName string, hcPort int32, isNodesHealthCheck bool) error {
+	// Prepare the firewall params for creating / checking.
+	desc := "Firewall for health checks from load-balancer."
```

Comment:
```
if isNodesHealthCheck
    use [makeFirewallDescription](https://github.com/kubernetes/kubernetes/blob/master/pkg/cloudprovider/providers/gce/gce_loadbalancer.go#L835)
else 
    {thoughts here?}
```

`"Firewall for health checks from load-balancer."` and `{"kubernetes.io/service-name":"%s", "kubernetes.io/service-ip":"%s"}` are vastly different. I'm wondering if we should just leave the common case description empty.